core-services/prow/02_config/openshift/library-go/_pluginconfig.yaml

```diff
@@ -1,3 +1,11 @@
+label:
+  restricted_labels:
+    openshift/library-go:
+    - allowed_users:
+      - xingxingxia
+      - wangke19
+      - gangwgr
+      label: cherry-pick-approved
```

Comment:
library-go is special. We also don't require backport-risk-assessed there. It is only a library and most PRs cannot be tested direction, without a proof PR in some other repo.